core/src/main/scala/org/http4s/multipart/MultipartParser.scala

```diff
@@ -318,7 +351,7 @@ object MultipartParser {
     } else if (currState == len) {
       splitCompleteMatch(middleChunked, currState, i, acc, carry, c)
     } else {
-      splitPartialMatch(middleChunked, currState, i, acc, carry, c)
+      splitPartialMatch0(middleChunked, currState, i, acc, carry, c)
```

Comment:
Sux but I had to do this because MiMa :(.

I messed up not making these private in the first place.